risc0/zkp/src/prove/soundness.rs

```diff
@@ -0,0 +1,292 @@
+use crate::{hal::Hal, taps::TapSet, FRI_FOLD, FRI_MIN_DEGREE, INV_RATE};
+use risc0_core::field::{baby_bear, ExtElem};
+
+/// Johnson's bound
```

Comment:
```suggestion
/// Johnson parameter. See https://eprint.iacr.org/2022/1216
```
m here is a parameter which controls proximity to the johnson bound